posts/inside-rust/2023-02-22-governance-reform-rfc.md

```diff
@@ -0,0 +1,7 @@
+# Governance Reform RFC Announcement Blog Post
+
+As part of [ongoing work on governance](https://blog.rust-lang.org/inside-rust/2022/10/06/governance-update.html), the "leadership chat" established a smaller "governance reform" working group to create an RFC to establish new project wide governance. This RFC is now live and can found on the RFCs repo: <https://github.com/rust-lang/rfcs/pull/NNNN>
```

Comment:
when were you planning to open that RFC?

Reply:
literally doing it right now with the rest of the wg sans mark